### PR TITLE
proper support for MultiplyWithElementOfCommutativeRingForMorphisms

### PR DESCRIPTION
**Pull request by mohamed-barakat**
and fixed a bug

### CODE REVIEW

File: GroupRepresentationsForCAP/gap/SemisimpleCategoryMorphisms.gi [hunk 1]
```diff
@@ -275,24 +275,11 @@ InstallMethod( Component,
...
-    
-end );
-
-##
+## shouldn't this simplify delegate to r * morphism?
 InstallMethod( \*,
                [ IsSemisimpleCategoryMorphism, IsRingElement ],
                
-  function( r, morphism )
+  function( morphism, r )
```
Comment (sebastianpos) [inline GroupRepresentationsForCAP/gap/SemisimpleCategoryMorphisms.gi]: Please delete this function as well.

This code was written when there was no primitive `AddMultiplyWithElementOfCommutativeRingForMorphisms`.
Now, the implementation of the 2-sided *-operator is automatic.

Comment (mohamed-barakat) [reply GroupRepresentationsForCAP/gap/SemisimpleCategoryMorphisms.gi]: I deleted the method.

**Comment by mohamed-barakat** [comment]
I deleted the obsolete methods.

**Comment by sebastianpos** [comment]
Thanks :-)

**Pull request merged**